src/auth/models.py

```diff
@@ -15,3 +15,21 @@ class UserSession(Base):
     computing_id = Column(
         String(32), nullable=False
     )  # technically a max of 8 digits https://www.sfu.ca/computing/about/support/tips/sfu-userid.html
+    # TODO: link to the user's table entry & remove the computing_id column (it already exists in the User table)
+
+
+class User(Base):
+    # user is a reserved word in postgres
+    # see: https://stackoverflow.com/questions/22256124/cannot-create-a-database-table-named-user-in-postgresql
+    __tablename__ = "site_user"
+
+    # note: a primary key is required for every database table
+    id = Column(Integer, primary_key=True)
+
+    # TODO: (#13) add two new columns for storing the initial date & last date logged in.
```

Comment:
@micahdbak you'll implement your feature here, use alembic (see https://github.com/CSSS/csss-site-backend/wiki/Alembic-%E2%80%90-Updating-the-DB-Schema), and add behaviour to the `/auth/login` endpoint. 

You'll also make your own `/auth/info` endpoint